charts/prometheus-pushgateway/templates/secret.yaml

```
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "prometheus-pushgateway.fullname" . }}
+  namespace: {{ template "prometheus-pushgateway.namespace" . }}
```

Could we use `include` instead of `template`? 

```suggestion
  namespace: {{ include "prometheus-pushgateway.namespace" . }}
```